integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -18,6 +18,7 @@
 
 package org.wso2.carbon.esb.passthru.transport.test;
 
+import junit.framework.Assert;
```

Comment:
Should we use TestNG assertions? Is there a usage for this?

Reply:
Removed the assertion. Please refer latest commit.